src/style.css

```diff
@@ -0,0 +1,25 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you can apply background `shorthand`

Reply:
I didn't know about this solution, thanks! :)